Sources/XRPKit/Models/XRPWallet.swift

```diff
@@ -36,12 +36,14 @@ public class XRPWallet {
     public var publicKey: String
     public var seed: String
     public var address: String
+    public var mnemonic: String
     
-    private init(privateKey: String, publicKey: String, seed: String, address: String) {
+    private init(privateKey: String, publicKey: String, seed: String, address: String, mnemonic: String = "") {
```

Comment:
Maybe we make mnemonic optional?

`mnemonic: String? = nil`

Reply:
👍

Reply:
With `mnemonic: String? = nil` I have to do something with self.mnemonic = mnemonic `Error is ?? or !`

I made a second option with another init and declaring the class var ? Is that better, idk, its passing.